### PR TITLE
Update design of extra building field

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Update design of extra building field.

Extract ToggleCard component (switch/checkbox).

## Justify why they are needed

We don't have design for this field yet, but it's a good start.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1186** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1186" target="_blank">Graphite</a> 
    * **PR #1187** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1187" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1187?utm_source=stack-comment).

**Review by guilhermespopolin — APPROVED** [review]
Nice :)

File: apps/store/src/components/PriceCalculator/ExtraBuildingsField.tsx [hunk 1]
```diff
@@ -173,27 +181,30 @@ const Card = styled(Space)(({ theme }) => ({
...
   alignItems: 'center',
-}))
+})
 
-const MutedText = styled.p(({ theme }) => ({
-  fontFamily: theme.fonts.body,
-  fontSize: theme.fontSizes[1],
+const MutedText = styled.span(({ theme }) => ({
   color: theme.colors.gray700,
 }))
```
Comment (KarnellSchultz) [inline apps/store/src/components/PriceCalculator/ExtraBuildingsField.tsx:209]: Could this be something like theme.text.secondary or use the ui component `<Text color="textSecondary" />`?

**Review by KarnellSchultz — APPROVED** [review]
LGTM

**Pull request merged**